packages/story-editor/src/components/panels/layer/constants.js

```diff
@@ -16,3 +16,5 @@
 
 export const LAYER_HEIGHT = 35;
 export const NESTED_PX = 30;
+
+export const nestedOffsetCalcFunc = (evt) => evt.offsetY < LAYER_HEIGHT / 1.7;
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick:**
Should this maybe go to an utils file since it doesn't seem to be a constant? 😄